package.json

```diff
@@ -1,40 +1,57 @@
 {
   "name": "posthtml-include",
-  "version": "1.1.0",
-  "description": "Include file in HTML",
-  "main": "index.js",
+  "version": "2.0.0",
+  "description": "Include Plugin for PostHTML",
+  "engines": { "node": ">=4" },
+  "main": "lib",
+  "scripts": {
+    "lint": "standard",
+    "test": "nyc ava -v test/index.js",
+    "logs": "standard-changelog >> CHANGELOG.md",
+    "docs": "jsdoc2md lib/*.js > INDEX.md",
+    "clean": "rm -rf .nyc_output coverage jsdoc-api dmd",
+    "start": "sudo npm run clean && npm run lint && sudo npm test"
+  },
   "dependencies": {
-    "posthtml-parser": "^0.1.1"
+    "read-cache": "^1.0.x"
   },
   "devDependencies": {
-    "chai": "^3.2.0",
-    "jscs": "^1.13.1",
-    "jshint": "^2.8.0",
-    "mocha": "^2.2.5",
-    "posthtml": "^0.8.1"
+    "ava": "^0.17.x",
+    "coveralls": "^2.x",
+    "jsdoc-to-markdown": "^2.x",
+    "nyc": "^10.x",
+    "posthtml-pug": "^1.0.x",
+    "posthtml-sugarml": "^1.0.0-alpha3",
+    "standard": "^8.x",
+    "standard-changelog": "0.0.1"
   },
-  "scripts": {
-    "test": "npm run lint && mocha",
-    "lint": "jshint . && jscs . -v",
-    "preversion": "npm test",
-    "postversion": "git push && git push --tags"
+  "files": [
+    "lib",
+    "LICENSE"
```

Comment:
The `LICENSE` file are always included, regardless of settings. See `files` section: https://docs.npmjs.com/files/package.json#files